core.js

```diff
@@ -47,6 +49,31 @@ Core.prototype.parsePixels = function (req, res, image) {
             let ansiColorCode = colorConvert.rgb.ansi16(pixel.r, pixel.g, pixel.b);
             writeStream.write('${AnsiColor.' + colors.ansiColorsMap[ansiColorCode] + '}', 'utf-8');
             writeStream.write('█', 'utf-8');
+            }
```

Comment:
Replace this two If with switch
default should log an error => "Unsupported mime type. Allowed types: jpg, png"